project-base/config/packages/shopsys_shop.yml

```diff
@@ -0,0 +1,3 @@
+parameters:
+    shopsys.router.locale_router_filepath_mask: '%shopsys.root_dir%/config/shopsys-routing/routing_front_*.yml'
```

Comment:
I was afraid of two folders routes and routing. This could be confusing.
The story behind this is – routes are loaded automatically, so every bundle installed can just add routing file into `routes` folder and it's done.
But `routing` (`shopsys-routing` at the moment) holds the configuration for SSFW specific routers – `FriendlyUrlRouter` and `LocalizedRouter`. These cannot be loaded automatically and have to be handled directly with the proper router.
And I was not able to come up with a better name for the folder without causing confusion.